test/unit/signal.spec.js

```diff
@@ -1,36 +1,167 @@
-import Signal from '../../lib/api/signal'
+import { MemoizedSignal, Signal } from '../../lib/api/signal'
 import {noop} from '../helpers'
 
-describe(`Signal`, () => {
-  describe(`constructor`, () => {
-    it(`creates a Signal instance`, () => {
+describe('MemoizedSignal', function () {
+  describe('constructor', () => {
+    it('creates a MemoizedSignal instance', () => {
+      expect(new MemoizedSignal(0)).to.be.instanceof(MemoizedSignal)
```

Comment:
Only exists since the spec had a test for this for old Signal implementation. Didn't want to remove stuff even though it seemed unnecessary to have it there.